src/Microsoft.Azure.WebJobs.Host/Triggers/TriggerArgumentBinding/SimpleTriggerArgumentBinding.cs

```diff
@@ -88,7 +89,10 @@ public virtual async Task<ITriggerData> BindAsync(TTriggerValue value, ValueBind
             string invokeString = await ConvertToStringAsync(eventData);
             var valueProvider = new ConstantValueProvider(userValue, this.ElementType, invokeString);
 
-            return new TriggerData(valueProvider, bindingData);
+            return new TriggerData(valueProvider, bindingData)
+            {
+                ReturnValueProvider = new ReturnValueProvider() { Value = "return" }
```

Comment:
Why is this needed? We shouldn't have to make any changes here.

Reply:
deleted.